### PR TITLE
feat: provide submit options for `mr create` and `issue create`

### LINKED ISSUE #329: Changes on current `mr create` and `issue create` flow
I had a couple of thoughts about the current `mr create` and `issue create` flow and came up with these issues and suggested changes.

**Issues**
1. Description step is not skippable for interactive mode
1. Users are unable to preview the issue or merge request body as can be done on the UI

**Suggested Changes**

- [x] Make the description step skippable

```sh
$ glab mr create
? Title This is a test issue title
? Description (nano) [Enter to launch editor, s to skip] 
```

- [x] Provide submit options to allow users either submit the MR via the CLI or `continue in browser` or `cancel`

```sh
$ glab mr create
? Title This is a test title
? Body (nano) <Received>
? Submit?  [Use arrows to move, type to filter]
> Yes
  Continue in browser
  Cancel
```

This will open the browser to a prefilled merge request with the title and body the user submitted. Similar flow for `issue create`. 

Note on Prefilling issue with url query parameters: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32449479

### PR DESCRIPTION
**Pull request by profclems**
Provide submit options to allow users either submit the MR via the CLI or continue in browser or cancel.

If 'preview in browser' is selected, it opens up a browser window with a prefilled title, description and
other mr or issue options.

Note on Prefilling issue with url query parameters: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32449479
and slash commands: https://gitlab.com/gitlab-org/gitlab-foss/-/issues/19731#note_32550046

Resolves #329

### CODE REVIEW

File: commands/issue/create/issue_create.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func NewCmdCreate(f *cmdutils.Factory) *cobra.Command {
 	}
 	issueCreateCmd.Flags().StringVarP(&opts.Title, "title", "t", "", "Supply a title for issue")
 	issueCreateCmd.Flags().StringVarP(&opts.Description, "description", "d", "", "Supply a description for issue")
-	issueCreateCmd.Flags().StringVarP(&opts.Labels, "label", "l", "", "Add label by name. Multiple labels should be comma separated")
+	issueCreateCmd.Flags().StringSliceVarP(&opts.Labels, "label", "l", []string{}, "Add label by name. Multiple labels should be comma separated")
```
Comment (maxice8) [inline commands/issue/create/issue_create.go]: How it works on the command line? We call --label multiple times?

Comment (profclems) [reply commands/issue/create/issue_create.go]: `--label` can be called multiple times. It also accepts comma-separated values in a string and converts them to slice

**Comment by maxice8** [comment]
I assume this will be merged today for the release ?

**Comment by profclems** [comment]
> I assume this will be merged today for the release ?

Yeah sure

**Comment by profclems** [comment]
Finally what I have been waiting for has been merged. I can do the release now https://github.com/ScoopInstaller/Main/pull/1640.
`glab` is now in the scoop main bucket

**Pull request merged**